src/main.js

```diff
@@ -1,32 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
`useEffect` is not being used at the component level. Plz remove it.

Reply:
Resolved.